src/relay/src/compiler/index.js

```diff
@@ -23,6 +23,7 @@ type ExternalOptions = {|
   +exclude: $ReadOnlyArray<string>,
   +validate: boolean,
   +watch: boolean,
+  +language: 'javascript' | 'typescript', // TODO: detect this automatically based on tsconfig.json file?
```

Comment:
I could imagine codebase with both...

Reply:
This is currently impossible to do with Relay Compiler as far as I know.

Reply:
I meant you could have both flowconfig and tsconfig.json in repo, because you previously used flow and then switched to typescript, just wild idea what's possible... :)

Reply:
OK, good point - no automatic detection then.